### PR TITLE
Bugfix/failingtests

### PR DESCRIPTION
**Pull request by danyball**
Fixes #

# Merge Checklist:
- [x] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [x] Tests are sufficient.
- [ ] Modifications still work in IE.
- [ ] Documentation is up to date.
- [ ] Changelog is up to date.
- [ ] Typescript typings for properties are up to date.
- [ ] Designers approved changes or no approval needed.
- [ ] Dependencies to other components still work.

### CODE REVIEW

File: src/components/20-molecules/datepicker/ui.test.js [hunk 1]
```diff
@@ -389,6 +373,25 @@ test('should highlight today', async t => {
   await t.expect(cell).eql('TODAY');
 });
 
+fixture('Datepicker without a valid startyear')
+  .page(
+    `${host}/iframe.html?id=components-datepicker--datepicker&knob-locale=de-CH&knob-year=x&knob-month=4&knob-day=22&knob-allowedyears=%5B%221971-2000%22%2C2012%2C2014%2C%222018-2022%22%5D&knob-label=&knob-monthtitle=Choose%20Month&knob-yeartitle=Choose%20Year&knob-invaliddatetext=Invalid%20date&knob-placeholder=Please%20select%20a%20date&knob-margin-top=0&knob-width%20%28not%20an%20attribute%29=&knob-disabled=true&viewMode=story`
```
Comment (markus-walther) [inline src/components/20-molecules/datepicker/ui.test.js]: very hard to see: knob-year=x. Either add a comment or use the power of string template literals to do:
`const ILLEGAL_YEAR = 'x'; .... &knob-year=${ILLEGAL_YEAR}...`

**Review by markus-walther — CHANGES_REQUESTED** [review]

File: src/components/20-molecules/datepicker/ui.test.js (hunk 1 above)
Comment (danyball) [reply src/components/20-molecules/datepicker/ui.test.js]: das problem bei dem Test ist, dass ich eig. gar kein year setzen will
aber das lässt storybook irgendwie nicht zu

File: src/components/20-molecules/datepicker/ui.test.js [hunk 2]
```diff
@@ -389,6 +373,26 @@ test('should highlight today', async t => {
   await t.expect(cell).eql('TODAY');
 });
 
+const illegal_year_to_avoid_storybook_default = 'x';
+fixture('Datepicker without startyear')
+  .page(
+    `${host}/iframe.html?id=components-datepicker--datepicker&knob-locale=de-CH&knob-year=${illegal_year_to_avoid_storybook_default}&knob-month=4&knob-day=22&knob-allowedyears=%5B%221971-2000%22%2C2012%2C2014%2C%222018-2022%22%5D&knob-label=&knob-monthtitle=Choose%20Month&knob-yeartitle=Choose%20Year&knob-invaliddatetext=Invalid%20date&knob-placeholder=Please%20select%20a%20date&knob-margin-top=0&knob-width%20%28not%20an%20attribute%29=&knob-disabled=true&viewMode=story`
```
Comment (markus-walther) [inline src/components/20-molecules/datepicker/ui.test.js:379]: now it's clear ;-)

**Review by markus-walther — APPROVED** [review]
LGTM

**Pull request merged**